github/gh-collaboration.qmd

```diff
@@ -21,6 +21,20 @@
 
 ## Code review
 
+- navigate to the top menu and click on `Pull requests`
+- by default all open pull requests are listed, you can further filter down the list. e.g. via clicking on `Reviews` and afterwards select `Awaiting review from you` in the drop down menu. This shows then only PRs where you are tagged as reviewer. 
+- click on a pull request of your choice 
+- the following window has 4 tabs: 
+  * Conversation: gives an overview about the PR
+  * Commits: list all commits of the PR
+  * Check:
+  * Files changed: lists all files which were modified 
+- click on the `Files changed` tab and go through the files and changes
+  * you can comment on a line by hovering over a line and click on the plus symbol 
+  * in case you want to comment on a block of lines: click and hold at the line number of the start of the block and release at the end of the block. Now you need to click on the plus symbol of the last line, in order to comment on the full block of lines

```

Comment:
It would be nice to add that no one sees these review comments until one clicks `Finish your review` if the Start a review button` is pressed and are marked as `pending`. If you `Add single comment` these are published immediately.

Reply:
sure. How should I solve this now? Do I need to Close this PR, make the changes in my repo and start from scratch (i.e. draft PR to Cormac, etc. ) ?

Reply:
No. You make the changes on your branch, and push the changes. They'll show up automatically on the PR.